var/spack/repos/builtin/packages/warpx/package.py

```diff
@@ -184,6 +189,9 @@ def cmake_args(self):
         with when('+openpmd'):
             args.append('-DWarpX_openpmd_internal=OFF')
 
+        if '+sensei' in spec:
+            args.append(self.define('SENSEI_DIR', os.path.join(spec['sensei'].prefix.lib, 'cmake')))
```

Comment:
Why do we need to set this? Aren't the usual [CMAKE_PREFIX_PATH](https://cmake.org/cmake/help/latest/variable/CMAKE_PREFIX_PATH.html) hints sufficient in Spack?

Reply:
I don't really understand why, but sometimes reading the CMAKE_PREFIX_PATH gets confused inside of the spack build environment. I have had to do this for a couple different projects but haven't spent too much time trying to understand why

Reply:
ok, makes sense

Reply:
Found the reason why this is needed:
Sensei does not yet install in the right standardized cmake paths. With 4.0, the config files land in `lib/cmake/` instead of `lib/cmake/sensei` on my Linux machine.
https://cmake.org/cmake/help/v3.24/command/find_package.html#config-mode-search-procedure